test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -353,6 +353,147 @@ public void TensorFlowTransformMNISTConvTest()
             }
         }
 
+        /// <summary>
+        /// The following python code was used to create tensorflow model.
+        /// 
+        /// from __future__ import print_function
+        ///
+        /// import tensorflow as tf
+        /// # Import MNIST data
+        /// from tensorflow.examples.tutorials.mnist import input_data
+        /// mnist = input_data.read_data_sets("/tmp/data/", one_hot = True)
+        ///
+        /// # Parameters
+        /// learning_rate = 0.01
+        /// training_epochs = 25
+        /// batch_size = 100
+        /// display_step = 1
+        ///
+        /// # tf Graph Input
+        /// x = tf.placeholder(tf.float32, [None, 784], name="Features") # mnist data image of shape 28*28=784
+        /// y = tf.placeholder(tf.float32, [None, 10], name = "Label") # 0-9 digits recognition => 10 classes
+        ///
+        /// # Set model weights
+        /// W = tf.Variable(tf.truncated_normal([784, 10]), name = "W")
+        /// b = tf.Variable(tf.constant(0.1, shape=[10], dtype=tf.float32), name = "b")
```

Comment:
>constant [](start = 31, length = 8)

can we verify that after the fine-tuning we are actually ending up with different values for the parameters ?

e.g. verifying the bias term we get after fine-tuning the model is different from the initialization value (0.1)

Reply:
Yes, I will do it.

---
In reply to: [221100728](https://github.com/dotnet/machinelearning/pull/1063#discussion_r221100728) [](ancestors = 221100728)